templates/tools/dockerfile/test/cxx_jessie_x64/Dockerfile.template

```diff
@@ -14,15 +14,15 @@
   # See the License for the specific language governing permissions and
   # limitations under the License.
 
-  FROM debian:jessie
+  FROM gcr.io/cloud-marketplace/google/rbe-debian8@sha256:496193842f61c9494be68bd624e47c74d706cabf19a693c4653ffe96a97e43e3
```

Comment:
nit: add an explanatory comment what's the image about - the image name is incomprehensible for someone who doesn't have lot of knowledge about foundry/RBE.

Reply:
Done in new Dockerfile